db/migrate/20181019161622_add_authors_column.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorsColumn < ActiveRecord::Migration[5.1]
+  def change
+    add_column :papers, :authors:, :text
```

Comment:
Looks like there's a small typo here:

```suggestion
    add_column :papers, :authors, :text
```